corehq/apps/app_manager/dbaccessors.py

```diff
@@ -121,3 +122,14 @@ def get_built_app_ids(domain):
     app_ids = [data.get('value', {}).get('copy_of') for data in result]
     app_ids = list(set(app_ids))
     return [app_id for app_id in app_ids if app_id]
+
+
+def get_exports_by_application(domain):
```

Comment:
I think the convention for `dbaccessors` is to put it in the same logical place as the couch view, so in this case the closest proxy would be `couchapps.dbaccessors`. it would be nice to extend that to be a package and add view-specific modules but seems out of scope for this PR